src/collector/watcher.ts

```diff
@@ -24,6 +24,16 @@ async function detectValidators(txs: Tx[]) {
   // extract validator addresses from string
   const addresses = uniq(
     txs
+      .filter((tx: any) => {
+        const msgs = tx?.value?.msg
```

Comment:
You can use the [nullish assignment](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Logical_nullish_assignment) operator and remove [line:30-32](https://github.com/terra-money/fcd/pull/118/files#diff-44a0fb2e0fe22052031f8c8b3aa35bb74636086e85897cc42539d6fc12211f2aR30-R32).


```js
const msgs = tx?.value?.msg ?? [];
```